RecoLocalMuon/GEMRecHit/python/me0RecHits_cfi.py

```diff
@@ -3,5 +3,5 @@
 me0RecHits = cms.EDProducer("ME0RecHitProducer",
     recAlgoConfig = cms.PSet(),
     recAlgo = cms.string('ME0RecHitStandardAlgo'),
-    me0DigiLabel = cms.InputTag("simMuonME0ReDigis"),
+    me0DigiLabel = cms.InputTag("simMuonME0PseudoReDigis"),
```

Comment:
can we settle on some common naming so that (local) reco configs do not get perturbed frequently just because of some internal DIGI-step reorganization?